src/lib/in.c

```diff
@@ -1378,15 +1394,6 @@ da2_cb(inputctx* ictx){
   return 2;
 }
 
-// weird form of Ternary Device Attributes used only by WezTerm
```

Comment:
i'm not sure i understand why you're getting rid of the wezterm handler?

Reply:
As @SirCyan said, it conflicted, and a review of the wezterm source history showed it never actually returned that.

I've updated the website for SyncTERM as well... hopefully it doesn't look as unmaintained anymore ;) (and v1.2rc1 was released today).